src/main.cpp

```diff
@@ -28,5 +29,7 @@ int main(int argc, char **argv)
     setenv("XDG_CACHE_HOME", appCachePath.toUtf8().constData(), 1);
 
     LunaQmlApplication application(argc, argv);
+
+    QtWebEngine::initialize();
```

Comment:
Is that necessary? Doesn't the QML plugin already do this? Or is this just to speedup things?